theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -5,28 +5,14 @@ set -xe
 echo "--Apt Hook Setup"
 
 pbuilder_hooksdir=/etc/pbuilder/${os}64/hooks
+local_hooksdir=$(PWD)/hooks-${os}-${version}-${BUILD_ID}
```

Comment:
or should this be `$(WORKSPACE)/hooks…`? I never know.

Reply:
I think `$WORKSPACE` is more predictable and helps guarantee the scope of this setup.